app/main/routes.py

```diff
@@ -30,30 +31,34 @@ def index():
 @main_bp.route('/login', methods=['GET', 'POST'])
 def login():
     title = "log in as a dog sitter"
-    if current_user.is_anonymous:
-        print("anonymous")
+    # if current_user.is_anonymous:
+    #     print("anonymous")
 
     if json_response_needed():
-        if current_user.is_authenticated:
-            print('current user')
-            print(current_user.first_name)
-            response = jsonify(current_user.to_dict())
-            response.status_code = 200
-            return response
+        # not using session for login or registrations 
+        # so remove current user from the code 
+        # bcoz that results in sending the same token 
+        # if current_user.is_authenticated:
+            # print('current user')
+            # print(current_user.first_name)
+            # response = jsonify(current_user.to_dict())
+            # response.status_code = 200
+            # return response
```

Comment:
can you remove these lines of code, it's not good practice to commit a lot of commented out code